agents/ocs_plugin_so.py

```diff
@@ -13,9 +13,8 @@
         ('Lakeshore370Agent', 'lakeshore370/LS370_agent.py'),
         ('Lakeshore240Agent', 'lakeshore240/LS240_agent.py'),
         ('Keithley2230G-PSU', 'keithley2230G-psu/keithley_agent.py'),
-        ('PysmurfController', 'smurf/pysmurf_control.py'),
+        ('PysmurfController', 'pysmurf_controller/pysmurf_controller.py'),
```

Comment:
Question about this, does this  determine what host-manager runs? I haven't been touching this file much but I guess I should.

Reply:
That's right, yes. Host Manager makes a call to [`ocs.site_config.scan_for_agents()`](https://ocs.readthedocs.io/en/develop/api.html?highlight=scan_for_agents#ocs.site_config.scan_for_agents), which uses this plugin file.